test/hpe_docker_unit_test.py

```diff
@@ -1,170 +1,207 @@
-import abc

```

Comment:
Please don't push this file. It has already been pushed by me and should be available post merge.

Reply:
Will it pass the CI nightly build if i remove from here and your code is still not merged?